web/backend/service/sc_memory/helpers/user.py

```diff
@@ -0,0 +1,112 @@
+from sc.core.transaction import (
+    TransactionNamesRead,
+    TransactionNamesWrite,
+    TransactionRead,
+    TransactionWrite,
+    FixedParameter,
+    AssignParameter,
+)
+from sc.core.types import (
+    ArcType,
+    LinkType,
+    NodeType,
+    TypeArcPerm,
+    TypeArcPos,
+    TypeConst,
+    TypeNodeStruct,
+)
+
+from .. import memory
+
+NodeConst = NodeType(const=TypeConst.CONST)
+NodeClass = NodeType(const=TypeConst.CONST, struct=TypeNodeStruct.CLASS)
+NodeNoRole = NodeType(const=TypeConst.CONST, struct=TypeNodeStruct.NO_ROLE)
+LinkConst = LinkType(const=TypeConst.CONST)
+ArcMemberConstPosPerm = ArcType(
+    const=TypeConst.CONST,
+    arc_pos=TypeArcPos.POS,
+    arc_perm=TypeArcPerm.PERM,
+    is_member=True,
+)
+
+
+def create_user_in_memory(user_id: int):
+    tr = TransactionNamesRead(memory.driver)
```

Comment:
I think we can remove this code for now. We will write that later with better API. It still now work correctly anyway